scripts/setup/configure-rabbitmq

```diff
@@ -5,11 +5,12 @@
 set -e
 set -x
 
+RABBITMQ_HOST=$("$(dirname "$0")/../get-django-setting" RABBITMQ_HOST)
 RABBITMQ_USERNAME=$("$(dirname "$0")/../get-django-setting" RABBITMQ_USERNAME)
 RABBITMQ_PASSWORD=$("$(dirname "$0")/../get-django-setting" RABBITMQ_PASSWORD)
-sudo rabbitmqctl delete_user "$RABBITMQ_USERNAME" || true
-sudo rabbitmqctl delete_user zulip || true
-sudo rabbitmqctl delete_user guest || true
-sudo rabbitmqctl add_user "$RABBITMQ_USERNAME" "$RABBITMQ_PASSWORD"
-sudo rabbitmqctl set_user_tags "$RABBITMQ_USERNAME" administrator
-sudo rabbitmqctl set_permissions -p / "$RABBITMQ_USERNAME" '.*' '.*' '.*'
+sudo rabbitmqctl -n "$RABBITMQ_HOST" delete_user "$RABBITMQ_USERNAME" || true
+sudo rabbitmqctl -n "$RABBITMQ_HOST" delete_user zulip || true
+sudo rabbitmqctl -n "$RABBITMQ_HOST" delete_user guest || true
+sudo rabbitmqctl -n "$RABBITMQ_HOST" add_user "$RABBITMQ_USERNAME" "$RABBITMQ_PASSWORD"
+sudo rabbitmqctl -n "$RABBITMQ_HOST" set_user_tags "$RABBITMQ_USERNAME" administrator
+sudo rabbitmqctl -n "$RABBITMQ_HOST" set_permissions -p / "$RABBITMQ_USERNAME" '.*' '.*' '.*'
```

Comment:
can you extract this as its own commit?  I sorta thought we'd done this, but my memory may be off.